packages/react-scripts/bin/react-scripts.js

```diff
@@ -43,6 +43,32 @@ switch (script) {
     process.exit(result.status);
     break;
   }
+  case 'lint': {
+    let eslintConfigPath;
+
+    try {
+      eslintConfigPath = require.resolve('eslint-config-react-app');
+    } catch (e) {
+      eslintConfigPath = require.resolve('../../../package.json');
+    }
+
+    const results = spawn(
+      'node',
+      [
+        require.resolve('eslint/bin/eslint'),
+        '--config',
+        eslintConfigPath,
+        'src/**/*.{js,jsx}',
```

Comment:
Need to include `.mjs` now that we support it.

It's interesting that this doesn't filter tests out. I guess we can say it's a feature. (Currently there's no way to lint tests.)